SessionUIKit/Utilities/String+SessionProBadge.swift

```diff
@@ -0,0 +1,40 @@
+// Copyright © 2025 Rangeproof Pty Ltd. All rights reserved.
+
+import UIKit
+
+public extension String {
+    enum SessionProBadgePosition {
+        case leading, trailing
+    }
+    
+    func addProBadge(
+        at postion: SessionProBadgePosition,
+        font: UIFont,
+        textColor: ThemeValue = .textPrimary,
+        proBadgeSize: SessionProBadge.Size,
+        spacing: String = " "
+    ) -> NSMutableAttributedString {
+        let image: UIImage = SessionProBadge(size: proBadgeSize).toImage()
```

Comment:
Mentioned earlier - might be nice to have a caching mechanism on `ImageDataManager` to avoid having to render this to an image over and over